python/catkin/builder.py

```diff
@@ -427,7 +427,7 @@ def build_catkin_package(
         make_executable = 'make'
 
     make_cmd = [make_executable]
-    make_cmd.extend(handle_make_arguments(make_args))
+    make_cmd.extend(handle_make_arguments(make_args, not use_nmake))
```

Comment:
Please pass keyword arguments by explicitly using the keyword: `handle_make_arguments(make_args, append_default_jobs_flags=not use_nmake)`.

Same below.